centreon/packages/ui/src/InputField/Select/Autocomplete/Connected/index.tsx

```diff
@@ -263,14 +263,6 @@ const ConnectedAutocompleteField = (
       }
     }, [optionsOpen]);
 
-    useEffect(() => {
-      setSearchParameter(
-        !isEmpty(searchConditions)
-          ? { conditions: searchConditions }
-          : undefined
-      );
-    }, [searchConditions]);
-
```

Comment:
I'd be very careful with this kind of changes, especially without proper tests... Are we sure that we don't introduce any regression?